rcode-reader/src/main/java/eu/rssw/pct/elements/BufferElement.java

```diff
@@ -0,0 +1,65 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.AccessibleClassElement;
+import eu.rssw.pct.RCodeInfo;
+
+public class BufferElement extends AccessibleClassElement {
+  private static final int DEFAULT = 1;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "DEFAULT" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1068)